mediator/med_map_mod.F90

```diff
@@ -272,6 +274,17 @@ subroutine med_map_routehandles_initfrom_field(n1, n2, fldsrc, flddst, mapindex,
        srcMaskValue = ispval_mask
        if (n1 == compocn .or. n1 == compice) srcMaskValue = 0
        if (n2 == compocn .or. n2 == compice) dstMaskValue = 0
+       if (n1 == compatm .and. n2 == compocn) then
+          if (trim(atm_name).eq.'datm') then
+             srcMaskValue = ispval_mask
```

Comment:
This was already set in line 274, no need to do it again.

Reply:
It was only in there for readability, we can remove it. We can get really technical and remove the trim because the fortran standard will pad the end of a string with " " until the two strings are of equal length. Are you okay with this change...
```
if (n1 == compatm .and. n2 == compocn) then
   if (atm_name.ne.'datm') then
      srcMaskValue = 1
   endif 
```

Reply:
I'm okay but for consistency I think the trim should be included.